lib/awful/tag.lua

```diff
@@ -1370,11 +1366,6 @@ object.properties(capi.tag, {
     setter_fallback = tag.setproperty,
 })
 
--- fix a load loop
-client = require("awful.client")
-capi.client.connect_signal("manage", function(c) client.object.to_selected_tags(c) end)
```

Comment:
Ok, then add the require to awful and ensure that `awful.rules` even tags a client if no rules match it.